charts/kargo/templates/controller/cluster-roles.yaml

```diff
@@ -44,13 +44,23 @@ rules:
 - apiGroups:
   - kargo.akuity.io
   resources:
-  - freights
   - promotions
   verbs:
   - create
   - get
   - list
   - watch
+- apiGroups:
+  - kargo.akuity.io
+  resources:
+  - freights
+  verbs:
+  - create
+  - get
+  - list
+  - watch
+  - patch
+  - update
```

Comment:
The controller requires permissions to patch the freight resource once the alias has been updated in order to reflect the updated value to the freight label.